pyspinw/symmetry/system.py

```diff
@@ -86,13 +102,60 @@ def violated_negative_constraints(self, unit_cell: UnitCell) -> list[str]:
                 out.append(name)
         return out
 
+    def validate(self, unit_cell: UnitCell):
+        """ Raises an exception if the unit cell is not compatible with this system """
+        # Positive constraints
+
+        for key, entry in self.positive_constraints.items():
+
+            if isinstance(entry, (int,float)):
+                value = entry
+            else:
+                value = value(unit_cell)

```

Comment:
I get an error here when I run the `examples/spacegroup_creation.py` script - I think it should be 

```
value = entry(unit_cell)
```